src/gateway/get-started/load-balancing.md

```diff
@@ -81,3 +83,12 @@ If you haven't completed these steps already, complete them before proceeding.
    Validate that the upstream you configured is working by visiting the route `http://localhost:8000/mock` using a web browser or CLI.
    Continue pinging the endpoint and the site should change from `httpbin` to `mockbin`.
 
+## What's next
+
+You've completed the Get Started with Kong guide, but a lot more is possible with [{{site.base_gateway}}](/gateway/latest). 
+The following are guides to advanced features of {{site.base_gateway}}:
+
+* [Monitoring with {{site.base_gateway}}](/gateway/3.0.x/production/monitoring/)
+* [Securing {{site.base_gateway}} with RBAC](/gateway/latest/kong-manager/auth/rbac/enable/) <span class="badge enterprise"></span> 
+* [Managing Workspaces and Team with {{site.base_gateway}}](/gateway/latest/kong-manager/auth/workspaces-and-teams/) <span class="badge enterprise"></span>
```

Comment:
Should the version for these links use `{{ page.kong_version }}` instead of hardcoded 3.0.x / latest?

Reply:
@Guaris mentioned to me either was allowed, but I'll modify if desired

Reply:
Let's switch. We shouldn't be using hardcoded links if it's possible to use the variable.

/latest/ is used in external content and anytime you're linking to a different top-level directory (eg from `/mesh/` to `/gateway/`, or from `/hub/` to anywhere); hardcoded versions are only used in the changelog.